infra/gcp/backup_tools/OWNERS

```diff
@@ -0,0 +1,14 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- cip-approvers
+- wg-k8s-infra-leads
+reviewers:
+- cip-reviewers
+- release-engineering-approvers
+- release-engineering-reviewers
```

Comment:
I used `k8s.gcr.io/images/k8s-staging-artifact-promoter/OWNERS` as the basis for other CIP-related stuff

Reply:
LGTM

Reply:
LGTM for CIP!